web-frontend/src/main/v3/packages/server-map/src/ui/ServerMap.tsx

```diff
@@ -10,7 +10,7 @@ import { keyBy } from 'lodash';
 
 cytoscape.use(dagre);
 
-type ClickEventHandler<T> = (param: { data?: T; eventType: 'right' | 'left'; position: cytoscape.Position }) => void;
+type ClickEventHandler<T> = (param: { data?: T; eventType: 'right' | 'left' | 'programmatic'; position: cytoscape.Position }) => void;
```

Comment:
type `ClickEventHandler` should probably be like this.  

```ts
type ClickEventHandler<T> = (param: {
  data?: T;
  eventType: 'right' | 'left';
  position: Partial<cytoscape.Position>;
}) => void;

```

Because of this https://github.com/pinpoint-apm/pinpoint/blob/b38493403610c830ccadddc40be19aed8854f584/web-frontend/src/main/v3/packages/server-map/src/ui/ServerMap.tsx#L257-L260

Reply:
Applied review